packages/fleather/README.md

```diff
@@ -1,7 +1,22 @@
-# Fleather 
-<!-- [![Build Status](https://travis-ci.com/memspace/zefyr.svg?branch=master)](https://travis-ci.com/memspace/zefyr) [![codecov](https://codecov.io/gh/memspace/zefyr/branch/master/graph/badge.svg)](https://codecov.io/gh/memspace/zefyr) -->
+[![Fleather & Parchment](https://github.com/fleather-editor/fleather/actions/workflows/fleather.yml/badge.svg)](https://github.com/fleather-editor/fleather/actions/workflows/fleather.yml)
+[![codecov](https://codecov.io/gh/fleather-editor/fleather/branch/master/graph/badge.svg?token=JRNFZ218FY)](https://codecov.io/gh/fleather-editor/fleather)
 
+# Fleather
 *Soft and gentle rich text editing for Flutter applications based on [Zefyr](https://github.com/memspace/zefyr).*
 
-[comment]: <> (![zefyr screenshot]&#40;https://github.com/memspace/zefyr/raw/master/packages/zefyr/zefyr.png&#41;)
+## Features
+* Inline attributes like **bold**, *italic*, ~~strikethrough~~ and etc.
+* Line attributes like direction, alignment, heading, number and bullet list and etc.
+* Block attributes like code, quote and etc.
+* Markdown-inspired semantics
+* Support for inline and block embeds
+* Support for markdown shortcuts
+* Using [Quill.js Delta](https://quilljs.com/docs/delta) as underlying data format by [Parchment](packages/parchment/README.md), Fleather is ready for collaborative editing using [OT](https://en.wikipedia.org/wiki/Operational_transformation) or [CRDT](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) (Not provided as built-in functionality yet)
```

Comment:
any idea how to do CRDT? I could easily part with OT :)

Reply:
No idea! I've never done OT or CRDT. @cgestes

Reply:
Yeah I think we can remove the CRDT part indeed
Never tried it and I'm not sure I intend to do so :)

Reply:
I am not sure the delta format allows for CRDT. I know for sure OT works :)